sn_node/src/node/messaging/join.rs

```diff
@@ -178,7 +178,8 @@ impl Node {
                 // Check if the joining node is younger than the youngest elder and older than
                 // MIN_ADULT_AGE in the first section, to avoid unnecessary churn during genesis.
                 let expected_age = FIRST_SECTION_MIN_ELDER_AGE - current_section_size as u8 * 2;
-                let is_age_invalid = peer.age() <= MIN_ADULT_AGE || peer.age() > expected_age;
+                let is_age_invalid =
+                    peer.age() <= MIN_ADULT_AGE || peer.age() > FIRST_SECTION_MIN_ELDER_AGE;
                 (is_age_invalid, expected_age)
```

Comment:
Did you mean to change this line as well?

```suggestion
                (is_age_invalid, FIRST_SECTION_MIN_ELDER_AGE)
```

If so, you can also remove line 180 as well

Reply:
No. the age is supposed to be still `changing`